charts/skypilot/templates/rbac.yaml

```diff
@@ -34,6 +34,17 @@ roleRef:
 {{- end }}
 {{- $namespaces = append $namespaces .Release.Namespace | uniq }}
 {{- range $namespaces }}
+{{- if and (ne . $.Release.Namespace) (not (lookup "v1" "Namespace" "" .)) }}
+---
```

Comment:
Can we add a quick comment that this condition basically iterates over `rbac.grantedNamespaces`? Better yet, can we make it an explicit loop on `rbac.grantedNamespaces` instead of having this `if` conditional?

Reply:
Good idea!

Reply:
I found it still need the non-equal check since user may accidentally include $.Release.Namespace in rbac.grantedNamespaces

Reply:
So maybe a comment is find